splunk_add_on_ucc_framework/global_config_update.py

```diff
@@ -123,9 +123,18 @@ def _handle_xml_dashboard_update(global_config: global_config_lib.GlobalConfig)
 
 def handle_global_config_update(global_config: global_config_lib.GlobalConfig) -> None:
     """Handle changes in globalConfig file."""
-    current_schema_version = global_config.schema_version
-    version = current_schema_version if current_schema_version else "0.0.0"
+    current_schema_version = global_config.schema_version or "0.0.0"
+
+    # TODO: update this variable at every schema version update
+    __ucc_latest_schema = "0.0.9"
```

Comment:
@lplonka-splunk @sgoral-splunk @kkedziak-splunk do you have an idea how can we simplify this? so we do not need to maintain this variable.

Reply:
I think we can do something like below:
```
if _version_tuple(global_config.meta.get("_uccVersion", "0.0.0")) >= _version_tuple("5.52.0"):
    # we hard-code the value of 5.52.0 as this feature would be shipped in that version
    version = current_schema_version
else:
    version = "0.0.0"
```

Reply:
`_uccVersion` suppose to be a build-only attribute, I would not try to bind to this variable.

Reply:
I do not want to block this PR only because of this change.